### PR TITLE
css fallbacks and code style fixes

### PR DESCRIPTION
**Pull request by J0WI**
- fixed fallbacks (standard syntax always at the end)
- fixed/added fallbacks for flexbox
- added quotes to url()

### CODE REVIEW

**Comment by Xenopathic** [comment]
What's the reasoning for the fallback and `url()` changes?

**Comment by J0WI** [comment]
fallback was wrong (prefix overwrote standard version, invalid flexbox properties), ```url()``` is code style. Quotes are used everywhere else (e.g. on inputs.css).

File: core/css/mobile.css [hunk 1]
```diff
@@ -10,15 +10,25 @@
 	-webkit-box-pack: center;
 	-webkit-box-align: center;
 
+	display: -webkit-flex;
```
Comment (MorrisJobke) [inline core/css/mobile.css]: The changes in this file are the only bits I'm worried about.

**Comment by MorrisJobke** [comment]
@owncloud/designers Please check out this PR.

File: core/css/mobile.css (hunk 1 above)
Comment (jancborchardt) [inline core/css/mobile.css]: Yep, not comfortable with those either. @J0WI can you separate them out and do another pull request for them? The others are only style fixes, but these here are functionality changes.

Comment (J0WI) [inline core/css/mobile.css]: Are you sure?
```display: -webkit-flex``` is just a newer syntax of ```display: -webkit-box```.
https://developer.mozilla.org/en-US/docs/Inbox/Advanced_layouts_with_flexbox#Flexbox_Containers

Comment (PVince81) [inline core/css/mobile.css]: Ok, then I guess it's fine. It looks like all this is just varied syntax for flexbox for different browsers.

**Comment by PVince81** [comment]
Can someone test this, especially mobile ? @owncloud/designers

**Comment by MorrisJobke** [comment]
I tested this on a 7 inch tablet, 5 inch smartphone (both with android - Chrome and Firefox), in Chrome and Safari and it looks good :+1: 

(I also tested stuff like the public page scroll with many files)


@PVince81 Mind to test this and review it :)

**Comment by MorrisJobke** [comment]
@owncloud/designers @nickvergessen @LukasReschke @rullzer Please review

**Comment by skjnldsv** [comment]
Seems good to me! 
_Chrome, ff & vivaldi (both mobile and desktop)._

**Comment by MorrisJobke** [comment]
> Seems good to me! 
> Chrome, ff & vivaldi (both mobile and desktop).

This could be counted as a 👍

**Comment by DeepDiver1975** [comment]
> Thanks a lot for your contribution!
> Contributions to the core repo require a signed contributors agreement http://owncloud.org/about/contributor-agreement/
> 
> Alternatively you can add a comment here where you state that this contribution is MIT licensed.
> 
> Some more details about out pull request workflow can be found here: http://owncloud.org/code-reviews-on-github/

@J0WI please respond to this - THX

**Comment by J0WI** [comment]
Since this document is only available in english I go with the MIT for this patch.

**Comment by DeepDiver1975** [comment]
> I'll go with the MIT license for this patch.

Thanks a lot!
